autosubmit/config/configcommon.py

```diff
@@ -203,10 +204,9 @@ def get_section(self, section, d_value="", must_exists=False):
                 "{0} must exists. Check that subsection {1} exists.".format(section_str, str(current_level)), 7014)
         if current_level is None or (not isinstance(current_level, numbers.Number) and len(current_level) == 0):
             return d_value
-        else:
-            return current_level
+        return current_level
 
-    def get_wchunkinc(self, section):
+    def get_wchunkinc(self, section: str) -> str:
```

Comment:
There are a few functions where I think we would have to change the code in order to being able to add types. For those functions, I left the code untouched as we will probably need good tests first. For other functions like this one I simply added the types as in pydocs or after reading the code.